src/main/webapp/index.jsp

```diff
@@ -38,7 +38,7 @@
                         <a class="mb-0 nav-link active" href="index.jsp"><i class="fas fa-home mr-2"></i>Home</a>
                     </li>
                     <li class="nav-item">
-                        <a class="mb-0 nav-link" href="post.jsp"><i class="fas fa-plus mr-2"></i>New Post</a>
+                        <a class="mb-0 nav-link" href="edit.jsp"><i class="fas fa-plus mr-2"></i>New Post</a>
```

Comment:
This linking doesn't seem to work for me. Maybe the wrong link?

Reply:
Yeah you are supposed to call PostEditServlet to load the post first before redirecting the user to edit.jsp, will fix that